ccm_web/server/src/api/api.section.handler.ts

```diff
@@ -4,10 +4,15 @@ import { APIErrorData } from './api.interfaces'
 import { handleAPIError, HttpMethod, makeResponse } from './api.utils'
 import { SectionUserDto } from './dtos/api.section.users.dto'
 import {
-  CanvasCourseSection, CanvasCourseSectionBase, CanvasEnrollment, CanvasEnrollmentWithUser, UserEnrollmentType
+  CanvasCourseSection,
+  CanvasCourseSectionBase,
+  CanvasEnrollment,
+  CanvasEnrollmentWithUser,
+  UserEnrollmentType
 } from '../canvas/canvas.interfaces'
 
 import baseLogger from '../logger'
+import { SectionExternalUserDto } from './dtos/api.section.external.users.dto'
```

Comment:
Nit-pick: I'd move this up to be next to the other `./dtos` import.

Reply:
Fixed.